src/components/ImageGrid/ImageGrid.vue

```diff
@@ -25,12 +25,12 @@
  * or display 'No More Media'.
  * Used to display both image search results, and related images.
  */
-import LoadMoreButton from '~/components/ImageGrid/LoadMoreButton'
+import VLoadMore from '~/components/VLoadMore'
```

Comment:
I think Krystle noted that we need to add `.vue` extension to the imports to use navigation to component in VS Code.